test.py

```diff
@@ -1,12 +1,202 @@
 import unittest
```

Comment:
Do you mean for all strings?

Reply:
Single quotes for string, yes, just to be more consistent. Some places it's double-quoted with single quotes inside it, which is fine.

Reply:
I changed all of the strings to use single quotes unless they are used as a key in a dictionary since those will need to be double-quoted in an f-string